cmd/utils.go

```diff
@@ -914,3 +914,21 @@ func listRedHatRegistryImageTags() {
 	}
 	parseMap(m, "tags", "registry.access.redhat.com/rhceph/rhceph-3-rhel7:")
 }
+
+func getImageName() string {
+	// If there is no configuration file, the imageName is the one from the command line
+	// If the imageName is still set to the default, let's return it immediately too
+	if (len(configurationFile) == 0) || (imageName == DEFAULTIMAGE) {
+		return imageName
+	}
+
+	// If there is a '-i' argument, let's check if the entry exists or return an error
+	if !isEntryExists(IMAGES, imageName) {
```

Comment:
What if the entry does not exist in the config file but I want to explicitly boot an image? without editing the config file. I don't think we should fail, we should try to boot anyway, if the image does not exist the registry will let us know.

Reply:
So if that name exists we use it from the config file unless we try directly to pull one named like that.

Reply:
Yes, so let's not fail.